filter_plugins/config_encoders.py

```diff
@@ -408,9 +408,14 @@ def encode_ini(
                 item = '""'
 
             if item is not None:
-                rv += "%s%s%s%s%s%s\n" % (
-                    indent, prop, delimiter, quote, _escape(item, quote),
-                    quote)
+                if item == "!!!null":
+                    rv += "%s%s\n" % (indent, prop)
+                else:
```

Comment:
Let's drop this part as it's very unlikely that somebody will ever need to set such value ;o)

Reply:
Removed now